.github/workflows/cd-syftcli.yml

```diff
@@ -94,3 +100,95 @@ jobs:
         run: |
           tox -e syftcli.publish
           twine upload packages/syftcli/dist/*
+
+  create-syftcli-binaries:
+    needs: [deploy-syft-cli]
+    if: always() && (needs.deploy-syft-cli.result == 'success' && ${{needs.deploy-syft-cli.outputs.current_hash != needs.deploy-syft-cli.outputs.previous_hash }}  || github.event.inputs.skip_tests == 'true')
+    strategy:
+      max-parallel: 99
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+      fail-fast: true
+    runs-on: ${{matrix.os}}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.SYFT_BUMP_TOKEN  }}
+
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.11"
+
+      - name: Install build dependencies for syftcli
+        run: |
+          pip install --upgrade pip
+          pip install -e ".[build]"
+        working-directory: ./packages/syftcli
+
+      - name: Install Tox
+        run: |
+          pip install -U tox
+
+      - name: Build syftcli
+        run: |
+          tox -e syftcli.build
+
+      - name: upload binaries
+        uses: actions/upload-artifact@v2
+        with:
+          name: syftcli-binaries
+          path: ./packages/syftcli/dist
+
+      # - name: Build syftcli for 'Linux'
+      #   if: matrix.os == 'ubuntu-latest'
+      #   run: |
+      #     pyinstaller --clean --onefile --name syftcli --distpath ./dist/cli src/syftcli/cli.py
+      #   working-directory: ./packages/syftcli
+
+      # - name: Build syftcli for 'MacOS'
+      #   if: matrix.os == 'macos-latest'
+      #   run: |
+      #     pyinstaller --clean --onefile --name syftcli --distpath ./dist/cli src/syftcli/cli.py --target-arch universal2
+      #   working-directory: ./packages/syftcli
+
+      # - name: Build syftcli for 'Windows'
+      #   if: matrix.os == 'windows-latest'
+      #   run: |
+      #     pyinstaller --clean --onefile --name syftcli --distpath ./dist/cli src/syftcli/cli.py
+      #   working-directory: ./packages/syftcli
+
+  create-syftcli-github-release:
+    needs: [create-syftcli-binaries]
+    if: always() && (needs.create-syftcli-binaries.result == 'success' || github.event.inputs.skip_tests == 'true')
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.SYFT_BUMP_TOKEN  }}
+
+      - name: Check if version is released on Pypi
+        id: check-pypi
+        run: |
+          VERSION = ${{needs.deploy-syft-cli.outputs.deployed_version}}
+
+          RESPONSE=$(curl -s --head https://pypi.org/pypi/syftcli/$VERSION/json)
+
+          if [[ $RESPONSE == *"200 OK"* ]]; then
```

Comment:
Q: I did a sample, testing with it, by below snippet

```sh
curl -s --head https://pypi.org/pypi/syftcli/0.1.3/json | grep "200 OK"
```

OK keywork is not in the received text, 

but the HTTP/2 200 is present

```sh
curl -s --head https://pypi.org/pypi/syftcli/0.1.3/json | grep "HTTP/2 200"
```